### ISSUE TITLE
GT: Placeholder survey for post meeting survey dev

### ISSUE THREAD

**Issue by adonahue** [rifflearning/zenhub-platform-project#249]
As a PM, I'd like to see an end-to-end user path for the GT project, so that it's clear all the pieces work together as expected, even if we still have some content details to fill in. 

** Acceptance Criteria **
- [ ] A placeholder post-meeting survey with a question about how the meeting went.
- [ ] Survey to be used in conjunction with story #225.

**Comment by jaedoucette** [comment]
If this is really just a placeholder (i.e. we can show _any_ survey), this is a 5 minute job once 225 is complete.

**Comment by jaedoucette** [comment]
@brecriffs @adonahue 

The link should be:

https://rc1userdr2g37fmjhl3w.sjc1.qualtrics.com/jfe/form/SV_09v0wmDlJORIbOt?UUID={UUID}

Important: Note that the UUID query param should be populated by Riff with the same UUID that the user signed in to Riff with. App.js populates the UUID and stores it in the fields of the user object.

**Comment by adonahue** [comment]
I think QA is blocked on this until @brecriffs hooks it together. I tried sticking one in the field but got an error. 

![image.png](https://images.zenhubusercontent.com/5cbf1c836caba0089f532d44/fe3cf65d-f329-4a4c-bc1b-7eb29364d7b5)

![image.png](https://images.zenhubusercontent.com/5cbf1c836caba0089f532d44/34938081-6b1d-44b1-b2e0-29b90d9ece08)

**Comment by adonahue** [comment]
@jaedoucette - any insight on the comments above? is there a way for me to test this directly in order to accept the story? Thanks!

**Comment by jaedoucette** [comment]
@adonahue I think this is likely blocked until Brec hooks it up. However, it _should_ accept a UUID in the login field (ideally, it should be passed automatically, and log you in automatically, but that's some stitching that'll happen when Brec adds the URL).

You might be able to test the survey itself by entering a valid UUID.

**Comment by adonahue** [comment]
ok, thanks @jaedoucette . I'll just wait until @brecriffs has stuff hooked up, which will be very soon anyhow. I did stick in another story (#255) that has the actual survey content if you want to just get the final version done. I'm also happy to fill it in, if you want to point me to the right place in Qualtrics.

**Comment by jaedoucette** [comment]
This issue (#249) has been superseded by #255, which is now in review & QA, so I'm going to close #249.
